src/hooks/useNotifications.tsx

```diff
@@ -82,24 +97,41 @@ const notificationQuery = gql`
               }
             }
           }
+          ... on PrivatePostNotification {
+            post {
+              id
+              __typename
+              ... on ActivePost {
+                authorId
+              }
+            }
+          }
         }
       }
     }
   }
 `;
 
-const supportedNotificationTypes = [
+export type SupportedNotifications = [
```

Comment:
If we move the type name to the notification types. This could become
`export type SupportedNotifications = FeedNotification['__typename'];`

Reply:
Done